packages/fxa-js-client/client/FxAccountClient.js

```diff
@@ -1018,6 +1024,83 @@ FxAccountClient.prototype.sessionStatus = function(sessionToken) {
     });
 };
 
+/**
+ * Verifies an account and a session using a short code that is based on otp.
+ *
+ * @method sessionVerifyCode
+ * @param {String} sessionToken User session token
+ * @param {String} code Code to be verified
+ * @param {Object} [options={}] Options
+ *   @param {String} [options.service]
+ *   Service being used
+ *   @param {Boolean} [options.marketingOptIn]
+ *   If `true`, notifies marketing of opt-in intent.
+ *   @param {Array} [options.newsletters]
+ *   Array of newsletters to opt in or out of.
```

Comment:
Is this strictly an opt-in?

Reply:
@shane-tomlinson Ack yea, you can not opt-out yet.